doc/api/net.md

```diff
@@ -416,109 +571,134 @@ See also: [`socket.setTimeout()`][]
 added: v0.1.90
 -->
 
-Returns the bound address, the address family name and port of the
-socket as reported by the operating system. Returns an object with
-three properties, e.g.
-`{ port: 12346, family: 'IPv4', address: '127.0.0.1' }`
+The `socket.address()` method returns the address to which the socket is
+bound, the address family name, and port of the socket as reported by the
+operating system.
+
+The object returned has the following properties:
+
+* `port` {number}
+* `family` {String} Either `'IPv4'` or `'IPv6'`
+* `address` {String} The IPv4 or IPv6 address
 
 ### socket.bufferSize
 <!-- YAML
 added: v0.3.8
 -->
 
-`net.Socket` has the property that `socket.write()` always works. This is to
-help users get up and running quickly. The computer cannot always keep up
-with the amount of data that is written to a socket - the network connection
-simply might be too slow. Node.js will internally queue up the data written to a
-socket and send it out over the wire when it is possible. (Internally it is
-polling on the socket's file descriptor for being writable).
+The read-only `socket.bufferSize` property specifies the current amount of data
+currently pending in the `net.Socket` instances write buffer.
+
+The `net.Socket` class has been designed such that calls to `socket.write()`
```

Comment:
seems to be describing things that are better described in the streams docs, this version will miss the warnings in https://github.com/nodejs/node/pull/10631